Makefile

```
@@ -0,0 +1,68 @@
+all: check coverage mutants
+
+.PHONY: \
+		all \
+		check \
+		clean \
+		coverage \
+		format \
+		install \
+		linter \
+		mutants \
+		results \
+		tests
+
+module = clean_enoa
+codecov_token = a6d7163e-a1c7-4a7c-98bc-a3ce0ea094a8
+
+define lint
+	pylint \
+        --disable=bad-continuation \
+        --disable=missing-class-docstring \
+        --disable=missing-function-docstring \
+        --disable=missing-module-docstring \
+        ${1}
+endef
+
+check:
+	black --check --line-length 100 ${module}
+	black --check --line-length 100 tests
+	flake8 --max-line-length 100 ${module}
+	flake8 --max-line-length 100 tests
+	mypy ${module}
+	mypy tests
+
+clean:
+	rm --force --recursive .*_cache
+	rm --force --recursive .tox
+	rm --force --recursive dist
+	rm --force --recursive ${module}.egg-info
+	rm --force --recursive ${module}/__pycache__
+	rm --force --recursive tests/__pycache__
+	rm --force .coverage
+	rm --force .mutmut-cache
+	rm --force coverage.xml
```

¿por favor podrías ordenar esto alfabéticamente?

```suggestion
	rm --force --recursive .*_cache
	rm --force --recursive .tox
	rm --force --recursive ${module}.egg-info
	rm --force --recursive ${module}/__pycache__
	rm --force --recursive dist
	rm --force --recursive tests/__pycache__
	rm --force .coverage
	rm --force .mutmut-cache
	rm --force coverage.xml
```

Otra opción:
```suggestion
	rm --force --recursive ${module}.egg-info
	rm --force --recursive ${module}/__pycache__
	rm --force --recursive .*_cache
	rm --force --recursive .tox
	rm --force --recursive dist
	rm --force --recursive tests/__pycache__
	rm --force .coverage
	rm --force .mutmut-cache
	rm --force coverage.xm
```

---

Me gusta más la segunda opción. Muchas gracias por las sugerencias